submissions/2025-09-02-tmQM-xtb-Dataset-T=100K-low-mw-high-coordinate-mult=3-v0.0/README.md

```diff
@@ -0,0 +1,61 @@
+# tmQM xtb Dataset T=100K low-mw high-coordinate mult=3 v0.0
+
+### Description
+
+This dataset was generated starting from the tmQM dataset (release 13Aug2024, https://github.com/uiocompcat/tmQM). 
+This dataset contains 6,425 unique systems with 206,240 total configurations / spin states below 600 Da.  The molecules are 
```

Comment:
Here, below in general information, and in the notebook dataset description -- I think this is actually 6905 as printed in the last NB cell?